internal/config/config.go

```diff
@@ -0,0 +1,16 @@
+package config
+
+import (
+	"github.com/kelseyhightower/envconfig"
+)
+
+type specification struct {
+	Port     int    `default:"5000"`
+	RedisUrl string `default:"redis://localhost:6379" envconfig:"REDIS_URL"`
+}
+
+var Config specification
+
+func GetConfig() error {
+	return envconfig.Process("tmpnotes", &Config)
```

Comment:
the env variables should be added to documentation on self-hosting.
TMPNOTES_REDIS_URL

Reply:
good idea - will add that

Reply:
@kwsorensen what do you think about this? https://github.com/soraro/tmpnotes/pull/34/commits/9955df0076f34c85b34f285cd3a055c1b801957a